### ISSUE TITLE
UnusedPrivateField violation even if a field is used in a nested class

### ISSUE THREAD

**Issue by Myllyenko** [CodeNarc/CodeNarc#121]
UnusedPrivateField violation triggers though a field is used in a nested classes.

**Comment by ychulovskyy** [comment]
@Myllyenko UnusedPrivateFieldRuleTest.testInnerClass() proves that the rule works correctly.
Could you please provide code example when it fails?

**Comment by Myllyenko** [comment]
```
final class Example
{
	private static final String FIELD = ''

	static final class NestedClass
	{
		String doSomething()
		{
			return Example.FIELD
		}
	}
}
```

Corresponding CodeNarc report:

[SRC]private static final String FIELD = ''
[MSG]The field FIELD is not used within the class Example

**Comment by ychulovskyy** [comment]
@Myllyenko Thanks for the inputs. I've create pull request to fix the issue.

**Comment by chrismair** [comment]
Fixed by #129
